qosf.org/mentorship_cohort_1.md

```diff
@@ -86,7 +87,7 @@ This is a showcase of all the projects created during the first cohort of our Qu
     
 *Mentee*: **[Eduard Smetanin](https://www.linkedin.com/in/eduard-smetanin-211b9972/)** is a Software Engineer with 15 years of experience with experience in various programming languages.
 
-*Mentor*: **[Ntwali Bashige](https://twitter.com/nbashige)** is a Quantum Software Engineer at [Zapata Computing](https://www.zapatacomputing.com/). 
```

Comment:
I have sent @ntwalibas a mail requesting his current employment info to add here.